nilearn/datasets.py

```diff
@@ -764,13 +796,12 @@ def fetch_craddock_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
     See http://www.nitrc.org/projects/cluster_roi/ for more information
     on this parcellation.
     """
-
     if url is None:
         url = "ftp://www.nitrc.org/home/groups/cluster_roi/htdocs" \
               "/Parcellations/craddock_2011_parcellations.tar.gz"
```

Comment:
> @AlexandreAbraham for consistency reasons, should we change 2011 to 2012 in nitrc too? I guess we'd want to keep the old (i.e. 2011) naming for a while for backward compatibility reasons ...

Does not matter - that is just some website. We should use the date when the paper in qeustion has been actually published in the physical world - and that is 2012. The delay is probably explained by the difference between paper accept and publish date.